contracts/V1/StateChannelDiamondProxy/StateSnapshotFacet.sol

```diff
@@ -105,7 +105,12 @@ contract StateSnapshotFacet is StateChannelCommon {
 
         _applyExitChannelBlocks(channelId, exitChannelBlocks);
 
+        // Update the state snapshot
         stateSnapshots[channelId] = lastProovenSnapshot;
+
+        // clear onChainSlashedParticipants
+        disputeData[channelId].onChainSlashedParticipants = new address[](0);
```

Comment:
Think it's better to use 'delete array' - gpt said it also clears the storage slots so you get a gas refund at the end of the tx

Reply:
thanks
done